Equipment/EquipmentDb/Gloves/plate.xml

```diff
@@ -228,6 +228,19 @@
 		</source>
 	</item>
 
+	<item id="22714" phase="4">
+		<info name="Sacrificial Gauntlets" type="PLATE" slot="GLOVES" unique="yes" req_lvl="60" item_lvl="68" quality="RARE" faction="HORDE" boe="no" icon="Inv_gauntlets_31.png"/>
```

Comment:
Remove the `faction` attribute, it is not restricted to either horde or alliance.

Set unique to `no` since it is not unique.